__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+
+beforeEach(() => {
+    return seed(data);
+  });
+
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+    test('GET: 200 - return all topics', () => {
+        return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics.length).toBe(3);
+        body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+    });
+
+    test('GET: 404 - invalid endpoint', () => {
+        return request(app)
+          .get('/api/authors')
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Not Found');
+          });
+      });
+});
```

Comment:
Excellent tests.
you may want to make the error test its own separate describe block though as this will apply to all incorrect endpoints and will save you from repeating it :)